src/main/java/org/apache/skywalking/banyandb/v1/client/BanyanDBClient.java

```diff
@@ -314,6 +317,38 @@ public void define(Measure measure) throws BanyanDBException {
         MetadataCache.INSTANCE.register(measure);
     }
 
+    /**
+     * Create or update the property
+     *
+     * @param property the property to be stored in the BanyanBD
+     */
+    public void save(Property property) throws BanyanDBException {
+        PropertyStore store = new PropertyStore(checkNotNull(this.channel));
+        try {
+            store.get(property.group(), property.name(), property.id());
+            store.update(property);
+        } catch (BanyanDBException ex) {
+            if (ex.getStatus().equals(Status.Code.NOT_FOUND)) {
```

Comment:
Is other API adopting status code later?

Reply:
The relevant querying APIs have a chance to follow this convention. I will fill an issue to track this.

Reply:
https://github.com/apache/skywalking/issues/8888 created, as no one created in 2 days.